hs_core/hydroshare/hs_bagit.py

```diff
@@ -106,10 +104,16 @@ def create_bag_files(resource):
 
     #Set properties of the aggregation
     a._dc.title = resource.title
-    a._dcterms.type = resource._meta.object_name
+    a._dcterms.type = URIRef(resource.metadata.type.url)
     a._citoterms.isDocumentedBy = metadata_url
     a._ore.isDescribedBy = res_map_url
 
+    res_type_aggregation = AggregatedResource(resource.metadata.type.url)
+    res_type_aggregation._rdfs.label = resource._meta.verbose_name
+    res_type_aggregation._rdfs.isDefinedBy = current_site_url + "/terms"
```

Comment:
@pkdash As long as this code is not run on www's resources before we do the migration, then this won't affect the migration.  +1.